packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -762,6 +763,9 @@ class ListTile extends StatelessWidget {
   // the default value to true.
   final bool internalAddSemanticForOnTap;
 
+  /// {@macro flutter.material.inkwell.statesController}
+  final MaterialStatesController? statesController;
```

Comment:
Ideally this would be WidgetStateController, but as it's used throughout the file as MaterialState, it's probably better for now that this is MaterialStatesController.